src/Proxy/CommonMethodProxy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Proxy;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\Proxy\ObjectProxy;
+use Yiisoft\Yii\Debug\Collector\CommonServiceCollectorInterface;
+
+class CommonMethodProxy extends CommonServiceProxy
```

Comment:
Why class has prefix `Common`. Do you plan additionally make non-common classes?

Reply:
Yes.

Reply:
May we avoid `Common`?

Reply:
I will think about that